pages/fellows/IvaKamie.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: fellow
+pagetype: fellow
+shortname: 82715941
+permalink: /fellows/82715941.html
+fellow-name: Ivan Prinko
+title: Ivan Prinko - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2022-07-06
+  end: 2022-09-14
+photo: /assets/images/team/Ivan-Prinko.jpg
+institution: Kyiv Academic University
+e-mail: ivan.prinko@protonmail.com
+project_title: Developing a Jupiter Notebook for converting confocal microscopic images to 3D cell images and metrics calculation.
+project_goal:
+    The project's objective is to provide an interactive tool for evaluating confocal microscopic images of cells in order to determine their features, including volume, polarity, and cross-sectional area.
+mentors:
+  - Abhishek Biswas (Princeton University)
+proposal: /assets/pdf/fellows-2022/<find-your-file>
```

Comment:
@IvaKamie this needs to be an actual file path. Can you open up another PR that corrects this to your proposal?